src/AppInstallerCLICore/Workflows/ValidateFlow.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ValidateFlow.h"
+
+namespace AppInstaller::CLI::Workflow
+{
+    void GetDependenciesFromManifest(Execution::Context& context) {

```

Comment:
All dependencies related functions should be in the dependencies files.  I get that the names of the files make it seem like they are related to the commands, but they are not tied to them.  I would much rather group them up by their relation to certain features, and dependencies is certainly large enough to warrant its own file.

Please move this one and the one from InstallFlow to the dependencies file.